CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -22,6 +22,79 @@ public class ReplicatorConfigurationRequestHandler {
         /////////////////////////////
             
         // TODO: Change client to expect replicator config, not the builder.
+        case "replicatorCollection_configure":
+            let conflictResolver: ConflictResolverProtocol? = args.get(name: "conflictResolver")
+            let pull_filter: Bool? = args.get(name: "pull_filter")!
+            let push_filter: Bool? = args.get(name: "push_filter")!
```

Comment:
```suggestion
            let enablePullFilter: Bool = args.get(name: "pull_filter")
            let enablePushFilter: Bool = args.get(name: "push_filter")
```

Get the value as `Bool` and not `Bool?` which might make it cleaner check below like 

```
if enablePullFilter {
 //// do 
}
```